### ISSUE TITLE
Installing to system Ghidra in Arch Linux needs workaround

### ISSUE THREAD

**Issue by b2ag** [zackelia/ghidra-dark#35]
Hi,

either it's not working out of the box with system wide Ghidra in Arch Linux or I simply don't understand the install process. It's unable to find Ghidra installation and when pointed to `/opt/ghidra/` it complains with `ERROR: Please open Ghidra at least once to fully install dark mode.`

Best

**Comment by ChongChengAC** [comment]
try:
```shell
sudo python install.py --path /opt/ghidra/ # ERROR: Please open Ghidra at least once to fully install dark mode.
# Then
python install.py --path /opt/ghidra/ # success
```
I don't know the reason but it works on Manjaro.

**Comment by b2ag** [comment]
Thanks a lot for the workaround. I'll leave this open until fixed I guess.

**Comment by ChongChengAC** [comment]
> Thanks a lot for the workaround. I'll leave this open until fixed I guess.

It  has been fixed but not merged into main. See #33

**Comment by zackelia** [comment]
Closed by #33.
